### PR TITLE
updated CommComm Chair status

### PR DESCRIPTION
**Pull request by WaleedAshraf**
Updated CommComm chair status as of https://github.com/nodejs/community-committee/issues/251

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,11 +25,10 @@ For all votes, a simple majority of all *Admin members* for, or against, the iss
 
 ### Contacts for assistance
 - [@ashleygwilliams](https://github.com/ashleygwilliams) - **Ashley Williams**, Individual Membership Director
-- [@bnb](https://github.com/bnb) - **Tierny Cyren**, CommComm Co-Chair
+- [@bnb](https://github.com/bnb) - **Tierny Cyren**, CommComm Chair
```
Comment (oe) [inline README.md]: just realized that this should say "Tierney"

Comment (WaleedAshraf) [reply README.md]: @oe Good catch 👍 fixed

**Comment by WaleedAshraf** [comment]
@bnb you may review.

**Comment by bnb** [comment]
LGTM. Going to merge this 👍

**Pull request merged**